test/cbmc/proofs/objectSearch/Makefile

```diff
@@ -19,6 +19,8 @@ USE_FUNCTION_CONTRACTS += skipCollection
 USE_FUNCTION_CONTRACTS += skipSpace
 USE_FUNCTION_CONTRACTS += skipString
 
+USE_EXTERNAL_SAT_SOLVER ?= --external-sat-solver cadical
```

Comment:
We still need to add a flag to our starter-kit to use `--sat-solver cadical` in a similar way we have `USE_EXTERNAL_SAT_SOLVER` instead of just passing it to `CBMCFLAGS`. Let's track this in our starter-kit side and update the proofs accordingly afterwards.